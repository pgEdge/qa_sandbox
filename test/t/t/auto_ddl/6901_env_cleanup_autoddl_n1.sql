-- This is a autoddl cleanup file cleaning up all objects created via 6001 setup script

DROP OWNED BY adminuser;

DROP OWNED BY appuser;

DROP FUNCTION IF EXISTS public.get_table_repset_info(TEXT);

DROP ROLE IF EXISTS appuser;

DROP ROLE adminuser;
