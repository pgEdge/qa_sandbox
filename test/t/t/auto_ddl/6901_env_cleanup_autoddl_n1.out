-- This is a autoddl cleanup file cleaning up all objects created via 6001 setup script
DROP OWNED BY adminuser;
INFO:  DDL statement replicated.
DROP OWNED
DROP OWNED BY appuser;
INFO:  DDL statement replicated.
DROP OWNED
DROP FUNCTION IF EXISTS public.get_table_repset_info(TEXT);
INFO:  DDL statement replicated.
DROP FUNCTION
DROP ROLE IF EXISTS appuser;
INFO:  DDL statement replicated.
DROP ROLE
DROP ROLE adminuser;
INFO:  DDL statement replicated.
DROP ROLE
